test/jdk/java/foreign/enablenativeaccess/panama_module/org/openjdk/foreigntest/PanamaMain.java

```diff
@@ -1,34 +0,0 @@
-/*
- * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
I noticed this file was unused, so I've remove it. It is superseded by `PanamaMainDirect`, but I guess we got both files through a merge somehow.